src/sentry/integrations/github/webhook.py

```diff
@@ -89,27 +100,33 @@ def _handle_delete(self, event, integration):
 
 class InstallationRepositoryEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent
-    def _handle(self, event, organization, repo):
+    def _handle(self, event, organization, repo, host=None):
         pass
 
 
 class PushEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#pushevent
+
     def is_anonymous_email(self, email):
         return email[-25:] == '@users.noreply.github.com'
 
     def get_external_id(self, username):
         return 'github:%s' % username
 
-    def get_client(self, event):
+    def get_idp_external_id(self, host=None):
+        # todo(meredith): when we have the integration will return
+        # integration.external_id
```

Comment:
this actually needs to lookup on the `{host}:{integration.metadata['app_id']`

Reply:
this one is for regular GitHub so it should use `options.get(github-app.id)` yeah?